www/caddy/Portfile

```diff
@@ -550,11 +550,11 @@ go.vendors          howett.net/plist \
                         rmd160  88f9577df93ac0f8801d8960adc7f28e38867f3e \
                         sha256  f69af10ff08c0e87f92dac3ee5172d8ed02463725b74edfc8943ef018a1a632d \
                         size    5343 \
-                    github.com/imdario/mergo \
```

Comment:
Keep the same name and add `repo github.com/darccio/mergo` after this.

Reply:
Looks like that works!